### ISSUE TITLE
Regarding SHWID resolution

### ISSUE THREAD

**Issue by themiszamani** [FC4E-WP5/metaresolver-handletemplate#13]
Dear all,
 
Currently SHWID resolution supports only the landingPage option.
By trying the following swhid: 21.T11973/MR@swh:1:cnt:94a9ed024d3859793618152ea559a168bbcbb5e2?landingpage
we get redirected to:
https://archive.softwareheritage.org/api/1/resolve/swh:1:cnt:94a9ed024d3859793618152ea559a168bbcbb5e2/
Which is not an actual landing page... rather an api documentation page that uses the specific pid as an example on how to make the api calls.
The real landing page could be the following which is using the archive domain name and the pid directly:
https://archive.softwareheritage.org/swh:1:cnt:94a9ed024d3859793618152ea559a168bbcbb5e2/
This automatically redirect us to: https://archive.softwareheritage.org/browse/content/sha1_git:94a9ed024d3859793618152ea559a168bbcbb5e2/ which allows us to browse the content and can be considered a landing page.
Since the specific pid represents content - a file - we can get the raw form of the file if we attach the `/raw` path to the end of the final location we were redirected at:
https://archive.softwareheritage.org/browse/content/sha1_git:94a9ed024d3859793618152ea559a168bbcbb5e2/raw
This can be be considered as resolving the resource itself.
As we've mentioned the https://archive.softwareheritage.org/api/1/resolve/swh:1:cnt:94a9ed024d3859793618152ea559a168bbcbb5e2/ call through the browser redirects us to an api documentation page. We can use the url parameter ?format=json to get the metadata of the object
e.g.
https://archive.softwareheritage.org/api/1/resolve/swh:1:cnt:94a9ed024d3859793618152ea559a168bbcbb5e2?format=json
which returns
```
{
  "namespace": "swh",
  "scheme_version": 1,
  "object_type": "content",
  "object_id": "94a9ed024d3859793618152ea559a168bbcbb5e2",
  "metadata": {
   
  },
  "browse_url": "https://archive.softwareheritage.org/browse/content/sha1_git:94a9ed024d3859793618152ea559a168bbcbb5e2/"
}
```
To sum up it seems to be possible to resolve
landingPage: https://archive.softwareheritage.org/swh:1:cnt:94a9ed024d3859793618152ea559a168bbcbb5e2/
metadata: https://archive.softwareheritage.org/api/1/resolve/swh:1:cnt:94a9ed024d3859793618152ea559a168bbcbb5e2?format=json
resource (when its swh:1:cnt):
https://archive.softwareheritage.org/browse/content/sha1_git:94a9ed024d3859793618152ea559a168bbcbb5e2/raw (and for the rest kind of swhids the resource is equal with the landing page)
What are your thoughts ?
Best
Themis

**Comment by asajedi** [comment]
@themiszamani The resoltion of SWHID is now implemented as documented above. Please check it via the MR UI and let me know if it works.

**Comment by themiszamani** [comment]
Thank you !!! Checking it NOW 👍
